src/applications/vaos/utils/appointment.jsx

```diff
@@ -522,49 +532,42 @@ export function getAppointmentAddress(appt, facility) {
   }
 
   if (facility) {
-    return `${facility.address.physical.address1} ${
-      facility.address.physical.city
-    }, ${facility.address.physical.state} ${facility.address.physical.zip}`;
+    return getFacilityAddress(facility);
   }
 
   return undefined;
 }
 
 /**
- * Function to generate ICS commands for an appointment.
+ * Function to generate ICS.
  *
- * @export
- * @param {*} appt
- * @param {*} facility
+ * @param {*} summary - summary or subject of invite
+ * @param {*} description - additional detials
+ * @param {*} location - address / location
+ * @param {*} startDateTime - start datetime in js date format
+ * @param {*} endDateTime - end datetime in js date format
  */
-export function generateICS(appt, facility) {
-  const cal = new ICS.VCALENDAR();
-  const event = new ICS.VEVENT();
-
-  const subject = getAppointmentTypeHeader(appt);
-  const description = `${getAppointmentInstructionsHeader(
-    appt,
-  )}. ${getAppointmentInstructions(appt)}`;
-  const location = getAppointmentAddress(appt, facility);
-
-  const duration = getAppointmentDuration(appt);
-  const startDateObj = getMomentConfirmedDate(appt).toDate();
-  const endDateObj = getMomentConfirmedDate(appt)
-    .add(duration, 'minutes')
-    .toDate();
-
-  cal.addProp('VERSION', 2);
-  cal.addProp('PRODID', 'VA');
-
-  event.addProp('UID');
-  event.addProp('SUMMARY', [subject]);
-  event.addProp('DESCRIPTION', [description]);
-  event.addProp('LOCATION', [location]);
-  event.addProp('DTSTAMP', startDateObj);
-  event.addProp('DTSTART', startDateObj);
-  event.addProp('DTEND', endDateObj);
-
-  cal.addComponent(event);
-
-  return cal.toString();
+
+export function generateICS(
```

Comment:
Takes care of https://github.com/department-of-veterans-affairs/va.gov-team/issues/5401, replacing use of `ics-js` since we are generating a very simple ICS.